assets/javascripts/wizard/initializers/custom.js.es6

```diff
@@ -16,6 +18,20 @@ export default {
     const autocomplete = requirejs('discourse/lib/autocomplete').default;
     const cook = requirejs('discourse/plugins/discourse-custom-wizard/wizard/lib/text-lite').cook;
     const Singleton = requirejs("discourse/mixins/singleton").default;
+    const WizardFieldDropdown = requirejs('wizard/components/wizard-field-dropdown').default;
+
+    WizardFieldDropdown.reopen({
+      tagName: 'span',
+      classNames: ['wizard-select-value'],
+
+      @computed
+      isFlagSelector() {
```

Comment:
@merefield We can make this one a later task, but tather than modifying the generic dropdown, we need to create a new flags-specific component which added to the flags plugin itself, in the same way the locations plugin adds a locations field to the custom wizard code, see: https://github.com/paviliondev/discourse-locations/tree/master/assets/javascripts/wizard

This approach, requiriing plugins integrating with the custom-wizard plugin to add components, rather than supporting those components in the custom-wizard itself is more sustainable over time. Now, we don't control the flags plugin, which is why this can be done later, but we need to make a task to do this and schedule it in. Could you add a note in the code for now, and create an open-source/task topic for this?

Reply:
Yeah, and I think he should probably take ownership of the .yml file?

Reply:
@merefield This is actioned right? (your PM on meta?)